generators/ionic/resources/oauth2/src/environments/environment.ts

```diff
@@ -1,20 +1,25 @@
+/* eslint-disable @typescript-eslint/naming-convention */
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+import type { IAuthConfig } from 'ionic-appauth';
+
 export const apiHost = 'http://localhost:8080/';
 
+const oidcConfig: IAuthConfig = {
+  client_id: 'D8kB8gP8ziVZHE8RIPaT33VtmmlHq6FO',
+  server_host: undefined,
+  redirect_url: window.location.origin + '/callback',
+  end_session_redirect_url: window.location.origin + '/logout',
+  scopes: 'openid profile',
+  pkce: true,
+};
+
 export const environment = {
   production: false,
   apiUrl: `${apiHost}api`,
-  oidcConfig: {
-    client_id: 'web_app',
-    server_host: 'http://localhost:9080/auth/realms/jhipster',
-    redirect_url: window.location.origin + '/callback',
-    end_session_redirect_url: window.location.origin + '/logout',
-    scopes: 'openid profile',
-    pkce: true,
-    audience: 'api://default'
-  },
+  oidcConfig,
+  audience: 'api://default',
   scheme: 'dev.localhost.ionic:/',
```

Comment:
```suggestion
  scheme: 'dev.localhost.ionic',
```
This change was made in https://github.com/jhipster/generator-jhipster-ionic/pull/673.